api/Makefile

```
@@ -1,4 +1,14 @@
-.PHONY: proto clean test build run deps dev migrate-up migrate-down docs
+.PHONY: proto sqlc clean test build run deps dev migrate-up migrate-down docs
+
+# Generate SQL code
+sqlc:
```

checkmake (0.2.2)

[warning] Target body for "sqlc" exceeds allowed length of 5 (7).

(maxbodylength)